scripts/protocgen.sh

```diff
@@ -1,39 +1,46 @@
 #!/usr/bin/env bash
+set -euo pipefail
 
-set -eo pipefail
+# By default, this script runs against the latest commit to the master branch
+# in the Tendermint spec repository. To use this script with a different version
+# of the spec repository, run it with the $VERS environment variable set to the
+# desired commit in the spec repo.
 
 : ${VERS:=master}
-URL_PATH=archive/
+URL_PATH=tarball/
 if [[ VERS -ne master ]]; then
-    URL_PATH=archive/refs/tags/v
+    URL_PATH=tarball/refs/tags/v
 fi
```

Comment:
I think these three lines and `URL_PATH` are no longer needed. The `commits/${VERS}` query works for branch names, tags, and commit hashes, and since you are resolving the branch to its SHA anyway, I think you can just hit: `https://api.github.com/repos/tendermint/spec/tarball/${REF}` unconditionally without path surgery.